addons/resource.language.en_gb/resources/strings.po

```diff
@@ -7379,7 +7379,7 @@ msgstr ""
 
 #: xbmc/settings/dialogs/GUIDialogAudioDSPManager.cpp
 msgctxt "#15040"
-msgid "Settings dialogue can be only used on OSD menu during playback!"
+msgid "The settings dialogue can only be used via OSD menu during playback!"
```

Comment:
To make it more clear this should probably be `The settings dialogue can only be opened from the OSD menu during playback`.